src/config.rs

```diff
@@ -88,11 +88,12 @@ impl Config {
     pub fn get_f64(&self, key: &str)-> Result<f64> {
         let value = self.get_element(key)?;
         Ok(value.as_f64()
-            .unwrap_or(
+            .or(
```

Comment:
Why this replacement works better?
If `value.as_f64()` returns `None`, we try to parse it as a integer (`value.as_i64()`) then try to unwrap the result (`.ok_or(...)?`) and finally cast it to `f64` (`as f64`). 
If `value.as_f64()` returns `Some(f64)` we just unwrap it to `f64` (`unwrap_or`).

Reply:
Discussed in private conversation that `unwrap_or` always calls an alternative branch (i.e. evaluates its argument) and throws an error from failing `?`. So we should either use `or` or somehow find a way to use `unwrap_or_else` (but I haven't found a way to do this here).
https://doc.rust-lang.org/std/option/enum.Option.html#method.unwrap_or